packages/graphiql/src/components/GraphiQL.tsx

```diff
@@ -21,13 +21,32 @@ import {
 } from 'graphql';
 
 import {
+  Button,
+  ButtonGroup,
+  ChevronDownIcon,
+  ChevronUpIcon,
```

Comment:
A tip, normally you don't need to have 2 same icons, you can use CSS for rotate your icon :)